e2e/cypress/integration/system_console/user_management_spec.js

```diff
@@ -147,6 +149,67 @@ describe('User Management', () => {
         });
     });
 
+    it('MM-T938 Users - Revoke a user\'s session', () => {
+        // # Create an active session for the user we're going to revoke.
+        apiLogin(testUser.username, testUser.password);
```

Comment:
this function is already exist
```cy.apiLogin({username: 'sysadmin', password: 'secret'});```

Reply:
> @mkraft
> these two test case are failing as user does not exist,
> Please see the screenshot, am I missing something ?
> 
> ![image](https://user-images.githubusercontent.com/6909890/101084592-2a7d3e80-357c-11eb-8d51-cd76ba4af9e2.png)

Can you try again with this latest update? It turns out some of the other tests update the email addresses that these new tests were looking for.